examples/experimental/dagster-airlift/dagster_airlift/constants.py

```diff
@@ -1,3 +1,3 @@
 MIGRATED_TAG = "airlift/task_migrated"
-DAG_ID_TAG = "airlift/dag_id"
-TASK_ID_TAG = "airlift/task_id"
```

Comment:
While we are doing this I think we should change the keyname to include the module name.

"dagster-airlift/dag_id"

I think using module name as a convention is a good one since it enforces global uniqueness.